js/radio.js

```diff
@@ -35,11 +35,13 @@
 		this.$radio = $(element).is('input[type="radio"]') ? $(element) : $(element).find('input[type="radio"]:first');
 		this.$label = this.$radio.parent();
 		this.groupName = this.$radio.attr('name');
-		this.$parent = this.$label.parent('.radio');
+		this.$blockWrapper = this.$label.parent('.radio');	// only used if block radio control, otherwise radio is inline
+		this.isBlockWrapped = true;	// initialized as a block radio control
 		this.$toggleContainer = null;
 
-		if (this.$parent.length === 0) {
-			this.$parent = null;
+		if (this.$blockWrapper.length === 0) {
+			this.$blockWrapper = null;
```

Comment:
I actually don't think we need to set this to null. I think that we were setting $parent to null to make boolean checking easier. That was a bad practice. Let's just leave $blockWrapper alone here...